natlas-server/run-server.sh

```diff
@@ -21,5 +21,5 @@ else
 	echo "$(date) : Production" >> logs/start.log
 	echo "$(date) : Production"
 
-	gunicorn -b 127.0.0.1:5000 natlas-server:app
+	gunicorn -b 0.0.0.0:5000 natlas-server:app
```

Comment:
If someone wants to run the server without docker, this is going to expose port 5000 on all ports. Is there a way around this so we don't need to change run-server.sh?

Reply:
I'm in agreement, I mentioned this in a DM before the PR got opened but it's a good idea to track it in ticket. Dockerizing should have as little impact on running it in a non-docker environment as possible.

Reply:
I can use bash param expansion to handle this. 

${BIND_ADDRESS:-127.0.0.1}

Reply:
Agreed. My first intention of this is just to get a repeatable dev-stack running. Then optimize for "production" and "security."

Reply:
This should be resolved in the latest commit.